app/src/main/java/openfoodfacts/github/scrachx/openfood/views/AddProductActivity.java

```diff
@@ -975,14 +975,15 @@ private void checkFields() {
         if (!editionMode) {
             if (addProductOverviewFragment.areRequiredFieldsEmpty()) {
                 viewPager.setCurrentItem(0, true);
-            } else if (isNutritionDataAvailable() && addProductNutritionFactsFragment.containsInvalidValue()) {
+            } else if (isNutritionDataAvailable() && addProductNutritionFactsFragment.isFieldsInvalid()) {
```

Comment:
weird naming: `is` but `fields` (plural or singular ?) . If the method checks if the fragment has an invalid value, then "hasInvalidValue" or "containsInvalidValue" is fine no ?

Reply:
Right, just wanted to uniform it to the other method. I'm renaming it.